packages/flame/lib/src/extensions/picture_extension.dart

```diff
@@ -0,0 +1,11 @@
+import 'dart:ui';
+
+extension PictureExtension on Picture {
+  /// Converts [Picture] into an [Image] and disposes of the picture.
+  Future<Image> asImage(int width, int height) {
```

Comment:
Aah, this is a good solution!

Reply:
I love this solution too! But I have problems with the method name, it is way too similar to `asImage`.

Maybe we could name this as `safeToImage` or something that makes it more clear what it does?